### ISSUE TITLE
[BUG] Get_variables Method

### ISSUE THREAD

**Issue by alejandrobarrio** [volanja/ansible_spec#71]
REF: https://github.com/volanja/ansible_spec/blob/master/lib/ansible_spec/load_ansible.rb#L314-L360

It's necessary use any recursive merge (a.k.a deep_merge) in order to solve problems like this:
            -> In roles/#{role}/defaults/main.yml

```
           var_a: val_a
           var_b:
                 var_b1: val_b1
                 var_b2: val_b2
                 var_b3: val_b3
```

   -> In #{inventory}/group_vars/all

```
          var_c: val_c
          var_b:
                 var_b4: val_b4
                 var_b5: val_b5
                 var_b6: val_b6
          var_d: val_d
```

The result must be: 

```
          var_a: val_a
          var_b:
                 var_b1: val_b1
                 var_b2: val_b2
                 var_b3: val_b3
                 var_b4: val_b4
                 var_b5: val_b5
                 var_b6: val_b6
          var_c: val_c
          var_d: val_d
```

But at this moment the result of this function is:

```
          var_a: val_a
          var_b:
                 var_b4: val_b4
                 var_b5: val_b5
                 var_b6: val_b6
          var_c: val_c
          var_d: val_d
```

**Comment by volanja** [comment]
@alejandrobarrio 
Is this solved this issue by #72 ?

if this issue was resolved, please close this issue.

**Comment by volanja** [comment]
This issue has been marked as stale because it has not had recent activity.
It will be closed if no further activity occurs.
Thank you for your contributions.
